packages/agent/src/utils/random.ts

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
Could you write some rationale here for why we are not requiring some well-established crypto library and always using it here?

```
import { randomBytes } from 'crypto';

export const randomNumber = (): number => {
  return randomBytes(4).readUInt32BE(0);
};
```

Reply:
Thanks!